src/components/ContentSection/ContentSection.tsx

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable react/no-danger */
+import React, { useContext, useEffect, useState } from 'react';
+import { Typography } from '@material-ui/core';
+import CreateIcon from '@material-ui/icons/Create';
+import { AuthContext } from '../../old/provider/AuthProvider/AuthContext';
+import { GeneralEditor } from '../Editor/GeneralEditor';
+import { VideoEditorToolbar } from '../Editor/Editors/VideoEditorToolbar';
+import { PostCreationButtons } from '../../views/postCreation/PostCreationButtons';
+import i18n, { langTokens } from '../../locales/localizationInit';
+import { useStyles } from './ContentSection.styles';
+import getDefContent from './defContent';
+
+export default function ContentSection(prop: { type: string }): JSX.Element {
+  const { authenticated } = useContext(AuthContext);
+  const [content, setContent] = useState('');
+  const [title, setTitle] = useState('');
+  const [edit, setEdit] = useState(false);
+
+  const openEditor = () => setEdit(true);
+  const closeEditor = () => setEdit(false);
+
+  const classes = useStyles();
+  const { type } = prop;
+
+  useEffect(() => {
+    switch (type) {
+      case 'about':
+        setContent(getDefContent(1));
```

Comment:
suggestion: numbers seem to just add unnecessary entities.

I would suggest using an object here as well